pyproject.toml

```diff
@@ -21,7 +24,10 @@ bandit = "^1.7.5"
 isort = "^5.12.0"
 axe-selenium-python = "^2.1.6"
 python-semantic-release = "<8.0.0"
-werkzeug = "2.3.7"
+urllib3 = "2.0.7"
+werkzeug = "2.3.8"
+cryptography = "42.0.5"
```

Comment:
Why is cryptography in dev dependencies?  You also have duplicates of urllib3, werkzeug and jinja2 between production and dev.  Can you explain what it is you need to do?